masterlist.yaml

```
@@ -15286,25 +15289,41 @@ plugins:
       - <<: *obsolete
         subs: [ '[Mage Equipment 21](https://www.nexusmods.com/oblivion/mods/18875/)' ]
         condition: 'checksum("Mage Equipment.esp", 27FA1E0F) or checksum("Mage Equipment.esp", 6693E84F)'
+      - <<: *patchIncluded
+        subs: [ 'Francesco''s Creatures and Items Mod' ]
+        condition: 'active("Francesco''s Leveled Creatures-Items Mod.esm") and not file("Mage Equipment addon - alter Fran NPCs.esp") and version("Mage Equipment.esp", "2.1.BlueSnake", >=)'
       - <<: *useVersion
         subs: [ 'Francesco''s Creatures and Items Mod' ]
-        condition: 'active("Francesco''s Leveled Creatures-Items Mod.esm")'
+        condition: 'active("Francesco''s Leveled Creatures-Items Mod.esm") and not version("Mage Equipment.esp", "2.1.BlueSnake", >=)'
     dirty:
       - <<: *quickClean
         crc: 0x60C24030
-        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
         udr: 23
+    clean:
+      - crc: 0x7A824244
+        util: 'TES4Edit v4.0.4c'
   - name: 'Francesco''s Mage Equipment.esp'
     dirty:
       - <<: *quickClean
         crc: 0xE8279E92
-        util: '[TES4Edit v4.0.4b](https://www.nexusmods.com/oblivion/mods/11536)'
+        util: '[TES4Edit v4.0.4c](https://www.nexusmods.com/oblivion/mods/11536)'
         udr: 23
   - name: 'Mage Equipment - No Manni.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/13695/' ]
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13695/'
+        name: 'Mage Equipment'
     msg:
       - <<: *obsolete
         subs: [ '[Mage Equipment 21](https://www.nexusmods.com/oblivion/mods/18875/)' ]
+  - name: 'Mage Equipment( addon - alter Fran NPCs)?\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/52290/'
+        name: 'Patch for Mage Equipment 21'
+  - name: 'Mage Equipment addon - alter Fran NPCs.esp'
```

It'd be helpful to mark the two Fran's plugins as incompatible:

```suggestion
  - name: 'Mage Equipment addon - alter Fran NPCs.esp'
    inc: [ 'Francesco''s Mage Equipment.esp' ]
```